liquibase-standard/src/main/java/liquibase/changelog/StatusChangeLogIterator.java

```diff
@@ -58,6 +61,18 @@ public void run(ChangeSetVisitor visitor, RuntimeEnvironment env) throws Liquiba
                         }
                     }
                 }
+                if(GlobalConfiguration.STRICT.getCurrentValue()) {
```

Comment:
Do we need this change to `StatusChangeLogIterator.java` ? It feels like everything could be validated in the Command Step as this code is executed before the status validation -> https://github.com/liquibase/liquibase/pull/6169/files#diff-d71f26aeddfbd41c01479ae9b2fd4ac32f42e76b8ca9c2b5bce22bb2260dbf47R108

Reply:
Done